src/main/java/com/openshift/restclient/IClient.java

```diff
@@ -30,6 +30,8 @@
 public interface IClient extends ICapable, IAuthorizationClient{
 	
 	IWatcher watch(String kind, String namespace, IOpenShiftWatchListener listener);
+
+	IWatcher watch(Collection<String> kinds, String namespace, IOpenShiftWatchListener listener);
```

Comment:
I usually like using something like  `IWatcher watch(String namespace, IOpenShiftWatchListener listener, String...kinds)` so you treat single/multiple params the same